src/index.js

```diff
@@ -25,6 +25,198 @@ function createPlugin() {
   const getRegistry = createGetRegistry();
   const once = createOnce();
 
+  function remarkPlugin(options = {}) {
```

Comment:
Can we just throw this into a separate file?

Reply:
Definitely, will do.